scenes/CompanyInfoScene.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import { View, Text } from 'react-native';
+
+export default class CompanyInfoScene extends Component {
+  static get defaultProps() {
```

Comment:
I'd prefer this to be outside of the class, in the form of
```javascript
CompanyInfoScene.defaultProps = {
...
}
```
but that's a matter of personal preference. You should be specifying prop types though.